springfox-spring-web/build.gradle

```diff
@@ -11,6 +11,7 @@ ext {
 
 dependencies {
   compile "org.reflections:reflections:$reflections"
+  compile files("${System.properties['java.home']}/../lib/tools.jar")
```

Comment:
I think that might be the better option. Plus you can control your timelines for release to suite your dependent teams. Once you think its ready we can and port it to the `springfox` organization as a separate library, that is of course, if you'd like to contribute it back for others to use. Would love to be a add-on library like the [grails integration](https://github.com/springfox/springfox-grails-integration)